src/families/ethereum/modules/send.js

```diff
@@ -59,9 +58,7 @@ const send: ModeModule = {
       }
 
       if (!t.data) {
-        if (!result.errors.amount && result.amount.eq(0)) {
-          result.errors.amount = new AmountRequired();
-        } else if (
```

Comment:
why would we want to send zero when data is not there? I guess some contract allows to do something like this? simple send without data i mean.

The current fix proposal is NOGO because it changes the behavior of Send flow and we don't want to do this, (introducing the possibility to send 0 on ETH). We have a long time history of doing that and rollbacking that (regarding UX of validation of send flow + many users would accidentally forget the amount, as we wouldn't block the NEXT).

I suggest a better fix is to have an explicit flag in the Transaction for you do say "allowZeroAmount" in context of wallet connect.

Reply:
ok, will do. i was expeting that this change would not be ok in its current form ^^